naga/src/back/msl/writer.rs

```diff
@@ -4005,6 +4005,13 @@ template <typename A>
     ) -> Result<(String, u32, u32), Error> {
```

Comment:
question: @gfx-rs/naga: Why do we not re-use the output of `wgpu_core::VertexFormat::size` here? 🤔 I feel like it should be the same in all cases, but I admittedly haven't analyzed for divergences.